src/InstrumentationEngine.NuGet.Headers/InstrumentationEngine.Nuget.Headers.csproj

```diff
@@ -2,13 +2,21 @@
 <!-- Copyright (c) Microsoft Corporation. All rights reserved.
      Licensed under the MIT License. -->
 <Project>
+  <PropertyGroup Condition="'$(Platform)'=='ARM64'">
+    <PackageId>Microsoft.InstrumentationEngine.Headers.arm64</PackageId>
+    <NuspecFile>InstrumentationEngine.Headers.arm64.nuspec</NuspecFile>
+  </PropertyGroup>
+  <PropertyGroup Condition="'$(Platform)'!='ARM64'">
+    <PackageId>Microsoft.InstrumentationEngine.Headers</PackageId>
+    <NuspecFile>InstrumentationEngine.Headers.nuspec</NuspecFile>
+  </PropertyGroup>
   <PropertyGroup>
     <AppendTargetFrameworkToOutputPath>false</AppendTargetFrameworkToOutputPath>
     <TargetExt>.nupkg</TargetExt>
-    <PackageId>Microsoft.InstrumentationEngine.Headers</PackageId>
     <CopyBuildOutputToOutputDirectory>false</CopyBuildOutputToOutputDirectory>
     <CopyOutputSymbolsToOutputDirectory>false</CopyOutputSymbolsToOutputDirectory>
     <GeneratePackageOnBuild>true</GeneratePackageOnBuild>
+    <Platforms>AnyCPU;x64;x86;arm64</Platforms>
```

Comment:
I think this should be `ARM64` to be consistent with the idea that Platform is set as upper case by is lowercase when used in file system naming.